SoftLayer/managers/hardware.py

```diff
@@ -297,7 +297,7 @@ def rescue(self, hardware_id):
         """
         return self.hardware.bootToRescueLayer(id=hardware_id)
 
-    def change_port_speed(self, hardware_id, public, speed):
+    def change_port_speed(self, hardware_id, public, speed, rebundant):
```

Comment:
redundant should default to None if not supplied.
`def change_port_speed(self, hardware_id, public, speed, redundant=None):`